### DISCUSSION TITLE
Строка с названиями праздников на каждый день

### DISCUSSION THREAD

**Discussion by kex29** [vvip-68/LedPanelWiFi#35] [Идеи и предложения]
Предлагаю организовать синхронизацию с календарем праздников на каждый день. Например, строка для сегодняшнего дня "21:19 06 декабря День лентяя"

**Comment by Oldroid123** [comment]
Может есть идеи по реализации?

**Comment by incorri** [comment]
Вопрос в том, откуда их брать, желательно по API
